wakeonlan/__init__.py

```diff
@@ -0,0 +1,14 @@
+from .wakeonlan import BROADCAST_IP
+from .wakeonlan import DEFAULT_PORT
+from .wakeonlan import create_magic_packet
+from .wakeonlan import main
+from .wakeonlan import send_magic_packet
+
+
+__all__ = (
+    "BROADCAST_IP",
+    "DEFAULT_PORT",
+    "create_magic_packet",
+    "send_magic_packet",
+    "main",
+)
```

Comment:
Do we need this? I think we either need to manually cherry-pick the imports above, or use `from .wakeonlan import *` and define `__all__` in `wakeonlan.py`, but not both.